routers/user.js

```diff
@@ -0,0 +1,62 @@
+const {getCurrentTimeYYYYMMDDSS} = require("../utility/time-utility");
+const {v4} = require("uuid")
+
+
+class User {
+    constructor(userCollection, dbService){
+        this.userCollection = userCollection
+        this.dbService = dbService;
+
+    }
+
+    async getMyUser(parm, context) {
+        // console.log(context.userInfo)
+
+        // await this.userCollection.insert(pipeline);
+        const user_email = {"$match": {
+            email: context.userInfo.email,
+            IDP: context.userInfo.IDP,}}
+        // const user_email = {"$match": {email: "testt@test.vom"}}
+        const sortCreatedAtDescending = {"$sort": {createdAt: -1}};
+        const limitReturnToOneApplication = {"$limit": 1};
+        const pipeline = [
+            user_email,
+            // user_idp,
+            sortCreatedAtDescending,
+            limitReturnToOneApplication
+        ];
+        let result = await this.userCollection.aggregate(pipeline);
+        let user
+        if (result.length < 1){
+            user = {
+                _id: v4(),
+                email: context.userInfo.email,
+                IDP: context.userInfo.IDP,
+                userStatus: "Active",
+                role: "User",
+                organizations: [],
+                firstName: context.userInfo.firstName,
+                lastName: context.userInfo.lastName,
+                createdAt: getCurrentTimeYYYYMMDDSS(),
```

Comment:
These times should be the same, create a time variable and call it once then use the time variable to populate both of these fields